### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Fynkzia**

### CODE REVIEW

File: src/arrayMethodJoin.js [hunk 1]
```diff
@@ -4,8 +4,22 @@
...
+  [].__proto__.join2 = function(separator = ',') {
+    let joined = '';
+
+    for (let i = 0; i < this.length - 1; i++) {
+      if (this[i] === null || this[i] === undefined) {
+        joined += `${separator}`;
+      } else {
+        joined += `${this[i]}${separator}`;
+      }
+    }
```
Comment (Nazar-Brunarskyi) [inline src/arrayMethodJoin.js:16]: you can create a separate variable for the value (`${separator}`), to not repeat yourself

**Review by Nazar-Brunarskyi — APPROVED** [review]
great job, just one tip)

File: src/arrayMethodJoin.js [hunk 2]
```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let joined = '';
```
Comment (solaryasha) [inline src/arrayMethodJoin.js:8]: What exactly was joined?

File: src/arrayMethodJoin.js [hunk 3]
```diff
@@ -4,8 +4,22 @@
...
+      if (this[i] === null || this[i] === undefined) {
+        joined += `${separator}`;
+      } else {
+        joined += `${this[i]}${separator}`;
+      }
+    }
+
+    if (this[this.length - 1] !== null && this[this.length - 1] !== undefined) {
+      joined += `${this[this.length - 1]}`;
+    }
```
Comment (solaryasha) [inline src/arrayMethodJoin.js:20]: Let's create a separate variable for the last element of the array.

**Review by solaryasha — CHANGES_REQUESTED** [review]

**Review by ivan-kadykalo — APPROVED** [review]
🚀

**Review by solaryasha — APPROVED** [review]

**Pull request closed without merging**